docs/src/main/tut/chisel3/interfaces-and-connections.md

```diff
@@ -83,6 +83,9 @@ Caution: bulk connections should only be used with **directioned elements** (lik
 ## The standard ready-valid interface (ReadyValidIO / Decoupled)
 
 Chisel provides a standard interface for [ready-valid interfaces](http://inst.eecs.berkeley.edu/~cs150/Documents/Interfaces.pdf).
+The ready-valid interface uses `ready` and `valid` to handshake
+data transfer. When both signals are asserted the data transfer takes
+place.
```

Comment:
I don't like `"handshake"` being used as a verb here. It also seems reasonable to be explicit about what each signal means and to document `fire`.

How about:

```markdown
A ready-valid interface consists of a `ready` bit, a `valid` bit, and some data stored in `bits`.
The `ready` bit indicates that a consumer is *ready* to consume data.
The `valid` bit indicates that a produced has *valid* data on `bits`.
When both `ready` and `valid` are asserted, a data transfer from producer to consumer takes place.
A convenience method `fire` is provided that is asserted if both `ready` and `valid` are asserted.
```

(GitHub really, really needs multi-line suggestions...)